vagrant/Vagrantfile

```diff
@@ -58,7 +58,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # View the documentation for the provider you're using for more
   # information on available options.
 config.vm.provider "virtualbox" do |v|
-  v.memory = 2048
+  #v.gui = true
+  v.memory = 8096
```

Comment:
Nice catch;) I did no thorough testing, but 2048 was too little at least. When trying anything else with that amount of memory whilst compiling, the OS would return an out-of-memory error. I believe 8192 would be the best choice, and hopefully not too demanding for todays laptops/desktops.